yb-voyager/cmd/import.go

```diff
@@ -173,17 +173,24 @@ func registerFFDBAsTargetConnFlags(cmd *cobra.Command) {
 
 func registerImportDataFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&disablePb, "disable-pb", false,
-		"true - to disable progress bar during data import (default false)")
+		"true - to disable progress bar during data import and stats printing during streaming phase (default false)")
 	cmd.Flags().StringVar(&tconf.ExcludeTableList, "exclude-table-list", "",
 		"list of tables to exclude while importing data (ignored if --table-list is used)")
 	cmd.Flags().StringVar(&tconf.TableList, "table-list", "",
 		"list of tables to import data")
-	cmd.Flags().Int64Var(&batchSize, "batch-size", -1,
+	defaultbatchSize := int64(DEFAULT_BATCH_SIZE_YUGABYTEDB)
```

Comment:
ok for now, in the interest of time, but we should revisit this -

Reply:
Maybe something similar for batchsize as well - https://github.com/yugabyte/yb-voyager/pull/1059#discussion_r1318270863

Reply:
hmm, I made this change for the import data help We should not mention the default as -1, because for users `import data` is YugabyteDB a specific command. Maybe for now this is fine but later we can register these flags separately for ff setup so that we can then mention ff db specific details without interfering with import msgs ?

Reply:
Got it. Okay 👍